docs/sandbox/VehicleParking.md

```diff
@@ -312,6 +312,66 @@ HTTP headers to add to the request. Any header key, value can be inserted.
 
 <!-- noi-open-data-hub END -->
 
+## Bikeep
+
+<!-- bikeep BEGIN -->
+<!-- NOTE! This section is auto-generated. Do not change, change doc in code instead. -->
+
+| Config Parameter                 |       Type      | Summary                                                                    |  Req./Opt. | Default Value | Since |
+|----------------------------------|:---------------:|----------------------------------------------------------------------------|:----------:|---------------|:-----:|
+| type = "vehicle-parking"         |      `enum`     | The type of the updater.                                                   | *Required* |               |  1.5  |
+| [feedId](#u__13__feedId)         |     `string`    | The name of the data source.                                               | *Required* |               |  2.2  |
```

Comment:
This "name of the data source" is repeated in parking updater docs, maybe call it `id` instead of `name`.

Reply:
I've updated it in https://github.com/opentripplanner/OpenTripPlanner/pull/5798/commits/091e4ed5620630e3f70e4eb585a733a2e28d4d2d 

Do you think that is a better way to explain it?